src/schnetpack/task.py

```diff
@@ -23,6 +23,7 @@ def __init__(
         loss_fn: Optional[nn.Module] = None,
         loss_weight: float = 1.0,
         metrics: Optional[Dict[str, Metric]] = None,
+        constraints: Optional[List[torch.nn.Module]] = None,
```

Comment:
add to __init__ docstring.

Also, the docstring of the class needs to be extended anyway (I will do that asap). Perhaps you can already add a small paragraph on how to use constraints?